src/data/__test-data.js

```diff
@@ -114,6 +115,39 @@ export const generateCMSHeaderBanner: ValueGenerator<mixed> = gen({
 });
 
 export const generateEvent: ValueGenerator<Event> = gen({
+  id: gen.alphaNumString,
+  contentType: "event",
+  locale: "en-GB",
+  revision: 1,
+  fields: gen({
+    name: "name",
+    eventCategories: ["Cabaret and Variety", "Music"],
+    audience: ["???"],
+    startTime: "2018-07-07T00:00+00:00",
+    endTime: "2018-07-07T03:00+00:00",
+    location: { lat: 0, lon: 10 },
+    addressLine1: "addressLine1",
+    addressLine2: "addressLine2",
+    city: "city",
+    postcode: "postcode",
+    locationName: "locationName",
+    eventPriceLow: 0,
+    eventPriceHigh: 10,
+    accessibilityOptions: ["accessibilityOptionsA", "accessibilityOptionsB"],
+    eventDescription: "eventDescription",
+    accessibilityDetails: "accessibilityDetails",
+    email: "email",
+    phone: "phone",
+    ticketingUrl: "ticketingUrl",
+    venueDetails: ["venueDetailsA", "venueDetailsB"],
+    individualEventPicture: generateFieldRef,
+    eventsListPicture: generateFieldRef,
+    performances: [],
+    recurrenceDates: []
```

Comment:
Do you think we should generate performances and recurrence dates? Or is this never used in tests?

Reply:
I've left these empty for now, but I have a task to fill in some of the blanks once we get the base of all these changes in. One thing I would like to do is refactor some of our tests to use the generators too. I think at that stage, filling these in becomes beneficial.